rustls/src/msgs/handshake.rs

```diff
@@ -1481,8 +1483,13 @@ impl CertificatePayloadTls13 {
 pub enum KeyExchangeAlgorithm {
     /// Key exchange performed via elliptic curve Diffie-Hellman.
     ECDHE,
+    /// Finite-field Diffie-Hellman Key exchange
```

Comment:
I think it's fine to call it `DHE`, but the rustdoc may should not call it FFDHE, I would suggest this:
```
/// Diffie-Hellman Key exchange with limited support on FFDHE (specified in )


```suggestion
    /// Diffie-Hellman Key exchange with limited support with only [FFDHE defined in RFC 7819].
    ///
    /// [FFDHE defined in RFC 7819]: https://datatracker.ietf.org/doc/html/rfc7919
```